### PR TITLE
[UX] More contrast for texts

### PR DESCRIPTION
**Pull request by andersonjeccel**
| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 

#### Description:

Based on user feedback:

> More contrast: replace the mid gray to a darker gray

The user is suggesting an improvement in the visual contrast of the interface by replacing mid-tone grays with darker shades. This indicates that the current color scheme may not provide sufficient contrast, potentially affecting readability and visual distinction between elements.

The user’s broader need is for a visually accessible and aesthetically pleasing interface that facilitates easy reading and navigation. They are seeking an enhancement in the visual hierarchy that allows users to quickly and effortlessly distinguish between different UI elements.

To address this feedback, this PR adjust the color palette to use darker grays for body text that offer better contrast against other elements. I ensured it meets WCAG accessibility standards for contrast ratios.

image

The contrast ratio was increased by about ~7 points compared to before.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Look at basically anything, the text will be darker

### CODE REVIEW

**Review by LordRembo — APPROVED** [review]
It's an easy fix to do (with no issues as consequence) with an immediate improvement for readability/accessibility, so sounds good to me!

File: app/bundles/CoreBundle/Assets/css/app.css [hunk 1]
```diff
@@ -1152,11 +1152,11 @@ h6,
   text-transform: uppercase;
 }
 .text-muted {
-  color: #8393a2;
+  color: #5d6c7c;
```
Comment (patrykgruszka) [inline app/bundles/CoreBundle/Assets/css/app.css:1168]: This change needs to be done in `CoreBundle/Assets/css/app/less/components/type.less:101`.

File: app/bundles/CoreBundle/Assets/css/app.css [hunk 2]
```diff
@@ -2912,18 +2912,18 @@ ul.line-legend li {
   background-color: #a0acb8;
 }
 .step > li.active .steps {
-  color: #47535f;
+  color: #262d33;
```
Comment (patrykgruszka) [inline app/bundles/CoreBundle/Assets/css/app.css:2943]: LESS file: `app/bundles/CoreBundle/Assets/css/app/less/components/steps.less`

File: app/bundles/CoreBundle/Assets/css/app.css [hunk 3]
```diff
@@ -3033,7 +3033,7 @@ ul.line-legend li {
 .bg-white {
   background-color: #fff !important;
   border-color: #f7f7f7 !important;
-  color: #47535f !important;
+  color: #262d33 !important;
```
Comment (patrykgruszka) [inline app/bundles/CoreBundle/Assets/css/app.css:3064]: LESS: `CoreBundle/Assets/css/app/less/components/colors.less`

**Review by patrykgruszka — CHANGES_REQUESTED** [review]
The `app.css` file should be generated automatically from the LESS files. You can generate the CSS by executing the `grunt compile-less` command.

**Comment by andersonjeccel** [comment]
@patrykgruszka Thanks for pointing!

As I changed 1 variable, every LESS rule or mixin based on the text color also changed accordingly and reflected on CSS when I ran `grunt compile-less`

These different hex codes are a consequence, nothing hardcoded 😉

_(have a look on the commit called “grunt compile-less” for reference)_

**Review by patrykgruszka — APPROVED** [review]
You're right. These variables are dependent on `@text-color`. My earlier comments were not accurate. Looks good 👍

**Review by RCheesley — APPROVED** [review]
Thanks for the PR! LGTM! 🚀

**Comment by andersonjeccel** [comment]
@RCheesley Based on the approved changes by Patryk and Rembrand and as there are no new commits since their review, can we consider as code-review-passed?

**Pull request merged**